cleanlab/object_detection/filter.py

```diff
@@ -0,0 +1,211 @@
+    _assert_valid_inputs,
+    get_label_quality_scores,
+    issues_from_scores,
+)
+
+
+def find_label_issues(
+    labels: List[Dict[str, Any]],
+    predictions: List[np.ndarray],
+    *,
+    return_indices_ranked_by_score: bool = False,
+) -> np.ndarray:
+    """
+    Identifies potentially mislabeled images in an object detection dataset.
+    An image is flagged with a label issue if *any* of its bounding boxes appear incorrectly annotated.
+    This includes images for which a bounding box: should have been annotated but is missing,
+    has been annotated with the wrong class, or has been annotated in a suboptimal location.
+
+    Suppose the dataset has ``N`` images, ``K`` possible class labels.
+    If ``return_indices_ranked_by_score`` is ``False``, a boolean mask of length ``N`` is returned,
+    indicating whether each image has a label issue (``True``) or not (``False``).
+    If ``return_indices_ranked_by_score`` is ``True``, the indices of images flagged with label issues are returned,
+    sorted with the most likely-mislabeled images ordered first.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors. 
+        This is a list of ``N`` dictionaries such that ``labels[i]`` contains the given labels for the `i`-th image in the following format:
+        ``{'bboxes': np.ndarray((L,4)), 'labels': np.ndarray((L,)), 'image_name': str}`` where ``L`` is the number of annotated bounding boxes
+        for the `i`-th image and ``bboxes[l]`` is a bounding box of coordinates in ``[x1,y1,x2,y2]`` format with given class label ``labels[j]``.
+        ``image_name`` is an optional part of the labels that can be used to later refer to specific images.
+
+       For more information on proper labels formatting, check out the [MMDetection library](https://mmdetection.readthedocs.io/en/dev-3.x/advanced_guides/customize_dataset.html)
+
+    predictions: 
+        Predictions output by a trained object detection model.
+        For the most accurate results, predictions should be out-of-sample to avoid overfitting, eg. obtained via :ref:`cross-validation <pred_probs_cross_val>`.
+        This is a list of ``N`` ``np.ndarray`` such that ``predictions[i]`` corresponds to the model prediction for the `i`-th image.
+        For each possible class ``k`` in 0, 1, ..., K-1: ``predictions[i][k]`` is a ``np.ndarray`` of shape ``(M,5)``,
+        where ``M`` is the number of predicted bounding boxes for class ``k``. Here the five columns correspond to ``[x1,y1,x2,y2,pred_prob]``,
+        where ``[x1,y1,x2,y2]`` are coordinates of the bounding box predicted by the model
+        and ``pred_prob`` is the model's confidence in the predicted class label for this bounding box. 
+
+        For more information on proper predictions formatting, check out the [MMDetection library](https://mmdetection.readthedocs.io/en/dev-3.x/advanced_guides/customize_dataset.html)
```

Comment:
I don't think this is the right link? This link is showing how to format the labels, not the predictions though

Reply:
@ulya-tkch It seems like you just removed the link for this?  That was not my intended suggestion, sorry for being unclear. I think it is good to link to MMDetection, you just need to link to where they describe how predictions are formatted.

Reply:
Unfortunately it seems MMDet does not describe how predictions are formatted at all. @aditya1503 spent a long time looking for any description and said he did not come across anything other than a slack post which I did not feel was good enough to link to. Should I add that?

Reply:
ah I see, maybe you can just say:

See the [MMDetection](https://github.com/open-mmlab/mmdetection) package for an example object detection library that outputs predictions in the correct format.

Reply:
(assuming this is 100% true, if some change is actually needed to reformat the predictions, then let's just not say anything and hope the tutorial is clear enough)